pkg/kubelet/pleg/generic.go

```diff
@@ -275,6 +288,15 @@ func (g *GenericPLEG) relist() {
 				metrics.PLEGDiscardEvents.Inc()
 				klog.ErrorS(nil, "Event channel is full, discard this relist() cycle event")
 			}
+			// Log exit code of containers when they finished in a particular event
+			if events[i].Type == ContainerDied {
+				containerID, ok := events[i].Data.(string)
+				if ok {
+					if exitCode, ok := containerExitCode[containerID]; ok {
+						klog.V(2).Infof("SyncLoop (PLEG+EC): container ID: %v, exit code: %v", containerID, exitCode)
```

Comment:
This isn't part of SyncLoop, so let's _not_ include `SyncLoop` in the message to avoid confusion. Also what is EC referring to?